src/spikeinterface/extractors/neoextractors/blackrock.py

```diff
@@ -117,6 +118,33 @@ def map_to_neo_kwargs(cls, file_path):
         neo_kwargs = {"filename": str(file_path)}
         return neo_kwargs
 
+    def build_stream_id_to_sampling_frequency_dict(self) -> Dict[str, float]:
+        """
+        Build a mapping from stream_id to sampling frequencies.
+
+        This function creates a dictionary mapping each stream_id to its corresponding sampling
+        frequency, as extracted from the signal channels in the Neo header.
+
+        Returns
+        -------
+        dict of {str: float}
+            Dictionary mapping stream_ids to their corresponding sampling frequencies.
+
+        Raises
+        ------
+        AssertionError
+            If there are no signal streams available from which to extract the sampling frequencies.
+        """
+        neo_header = self.neo_reader.header
+        if "spike_channels" in neo_header and neo_header["spike_channels"].size > 0:
+            channels = neo_header["spike_channels"]
+            channel_sampling_frequencies = channels["wf_sampling_rate"]
```

Comment:
In neo this is not necessary the sampling frequency of the spikes (even it is very often the same).
This is the frequency of the waveforms snippet. And this field has no garnty to be here if the snipet are not recorded

Reply:
yeah, I guess for now we're going with the fixed sampling rate